integration-tests/utils/__init__.py

```diff
@@ -1,35 +1,34 @@
+import pytest
 import sh
 
 
-def yggdrasil_service_is_active():
-    """Method to verify if yggdrasil is in active/inactive state
-    :return: True if yggdrasil in active state else False
+def service_is_active():
```

Comment:
I would not rename this method (I would keep `yggdrasil_service_is_active`), because there is also rhsm.service that is used by rhc. It could be confusing what service is mentioned here. It is OK to keep upstream name yggdrasil for methods and variables.